src/s2client-fsharp/Sc2Game.fs

```diff
@@ -2,26 +2,36 @@ namespace Starcraft2
 
 open SC2APIProtocol
 
-
 type GameState =
-  { LastObservation : SC2APIProtocol.ResponseObservation option
-    LastActions : SC2APIProtocol.Action list
-    NewObservation : SC2APIProtocol.ResponseObservation
-    // more global state
-    PlayerId : PlayerId
-    GameInfo : SC2APIProtocol.ResponseGameInfo
-     }
-    static member Empty playerId = 
-      { LastObservation = None
-        LastActions = []
-        NewObservation = null
-        PlayerId = playerId
-        GameInfo = null }
+    { 
```

Comment:
I'd prefer the recommended style from https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-record-declarations.